### PR TITLE
luaradio: new port, version 0.6.1

### PR DESCRIPTION
**Pull request by vsergeev**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.3
Xcode 11.3.1

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: science/luaradio/Portfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+categories          science comms
+platforms           darwin
+license             MIT
+maintainers         {@vsergeev sergeev.io:v}
+description         A lightweight, embeddable software-defined radio framework built on LuaJIT
+long_description    ${description}
+homepage            https://luaradio.io/
+
+version             0.6.1
+github.setup        vsergeev luaradio ${version} v
```
[the remark this answers is not in the document]
Comment (mf2k) [reply science/luaradio/Portfile]: Also, the github.setup line is just above the `name` line. 

In fact, if you put it there, the `name` line is also redundant and should be removed.

File: science/luaradio/Portfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                luaradio
+categories          science comms
+platforms           darwin
+license             MIT
+maintainers         {@vsergeev sergeev.io:v}
```
Comment (mf2k) [inline science/luaradio/Portfile]: Can `openmaintainer` be added?

**Comment by vsergeev** [comment]
Thanks for the feedback @mf2k, I've updated the Portfile with the requested changes.

I would just add as a comment or observation that it's surprising to me that fields like `name` and `version` aren't explicitly required in general. I see now that the github PortGroup will specify them for you, but this implies that the choice of standard fields to provide or to omit depends on specific knowledge of the PortGroup being used, while I expected the github PortGroup would behave more like a fetch helper extension to the standard package fields.

**Comment by mf2k** [comment]
Let's give time to other committers to review this.

**Comment by vsergeev** [comment]
Any updates on this? -- would love to have this merged so I could close some PRs on my end.

**Comment by vsergeev** [comment]
Thank you!

**Pull request merged**